dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/Collections/CollectionConformanceTests.cs

```diff
@@ -8,136 +8,92 @@
 
 namespace VectorDataSpecificationTests.Collections;
 
-public abstract class CollectionConformanceTests<TKey>(VectorStoreFixture fixture)
+public abstract class CollectionConformanceTests<TKey>(VectorStoreFixture fixture) : IAsyncLifetime
     where TKey : notnull
 {
+    public Task InitializeAsync()
+        => fixture.VectorStore.DeleteCollectionAsync(this.CollectionName);
+
     [ConditionalFact]
-    public async Task VectorStoreDeleteCollectionDeletesExistingCollection()
+    public async Task Collection_Ensure_Exists_Delete()
```

Comment:
I am sorry, but I could not resist:

![image](https://github.com/user-attachments/assets/0c5fbc7e-37e1-496b-9243-29eb2b229930)